packages/edit-post/src/components/welcome-guide/images.js

```diff
@@ -0,0 +1,36 @@
+		{ ...props }
+	/>
+);
+
+export const InserterIconImage = ( props ) => (
+	<img
+		alt={ __( 'inserter' ) }
+		src="data:image/svg+xml;charset=utf8,%3Csvg width='18' height='18' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill-rule='evenodd' clip-rule='evenodd' d='M8.824 0C3.97 0 0 3.97 0 8.824c0 4.853 3.97 8.824 8.824 8.824 4.853 0 8.824-3.971 8.824-8.824S13.677 0 8.824 0zM7.94 5.294v2.647H5.294v1.765h2.647v2.647h1.765V9.706h2.647V7.941H9.706V5.294H7.941zm-6.176 3.53c0 3.882 3.176 7.059 7.059 7.059 3.882 0 7.059-3.177 7.059-7.06 0-3.882-3.177-7.058-7.06-7.058-3.882 0-7.058 3.176-7.058 7.059z' fill='%234A4A4A'/%3E%3Cmask id='a' maskUnits='userSpaceOnUse' x='0' y='0' width='18' height='18'%3E%3Cpath fill-rule='evenodd' clip-rule='evenodd' d='M8.824 0C3.97 0 0 3.97 0 8.824c0 4.853 3.97 8.824 8.824 8.824 4.853 0 8.824-3.971 8.824-8.824S13.677 0 8.824 0zM7.94 5.294v2.647H5.294v1.765h2.647v2.647h1.765V9.706h2.647V7.941H9.706V5.294H7.941zm-6.176 3.53c0 3.882 3.176 7.059 7.059 7.059 3.882 0 7.059-3.177 7.059-7.06 0-3.882-3.177-7.058-7.06-7.058-3.882 0-7.058 3.176-7.058 7.059z' fill='%23fff'/%3E%3C/mask%3E%3Cg mask='url(%23a)'%3E%3Cpath fill='%23444' d='M0 0h17.644v17.644H0z'/%3E%3C/g%3E%3C/svg%3E"
+		{ ...props }
+	/>
+);
```

Comment:
Did you check the impact on the bundle size of the script?

Reply:
`master`:

```
./build/nux/index.js    7.46 KiB      34  [emitted]         nux
```

`add/welcome-guide`:

```
 ./build/nux/index.js    10.9 KiB      34  [emitted]         nux
```

So an extra 3.44 KB. What do you think? Happy to move them somewhere else.

I was thinking ideally I'd like to have a SVG loader set up in Webpack so that we can `import()` it in a lazy fashion. Maybe I can add a note to https://github.com/WordPress/gutenberg/issues/2768.

Reply:
Taking about `edit-post` package.

Reply:
🤦‍♂️ ich bin dumm.

`master`:

```
./build/edit-post/index.js    83.9 KiB      20  [emitted]         editPost
```

`add/welcome-guide`:

```
./build/edit-post/index.js    98.2 KiB      20  [emitted]         editPost
```

14.3 KB.

Reply:
That's less than I expected, so I think it's acceptable.

Reply:
je bent een ster